app/controllers/patients_controller.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+# Define the patients controller class.
+class PatientsController < ApplicationController
+  include AccessHelper
+
+  before_action :confirm_logged_in
+
+  # GET /patients
+  # Display patients list.
+  def index
+    @patients = Patient.sorted_by_id
+  end
+
+  # GET /patients/:id
+  # Display patient information.
+  def show
+    @patient = Patient.find(params[:id])
```

Comment:
I think that you can do def show; here too

Reply:
How would I get the patient info in the view if I just do def show;?

Reply:
rails magic 💯